standalone/RL_plus_script.py

```diff
@@ -204,21 +207,23 @@ def get_action_sequence():
 
 
 def test():
-    env = gym.make('MineRLObtainDiamond-v0')
+    action_sequence = get_action_sequence()
+    writer = SummaryWriter(f"runs/{experiment_name}")
+    env = gym.make('MineRLObtainDiamond-v0').env
+    env = gym.wrappers.TimeLimit(env, config["TREECHOP_STEPS"])
```

Comment:
Hmm I am not sure if this is same behaviour as rest of the code (the env itself contains a timelimit). The TREECHOP_STEPS is used to split the logic into two: first do at most TREECHOP_STEPS amount of steps to obtain logs, and then move onto other stuff (e.g. crafting etc)

Reply:
So why would that be an issue? You have the `env1` to do the stuff after `TREECHOP_STEPS`

Reply:
I am worried that it could force a reset too early with this setup (I have not tested it, but I think behaviour would change). Is there reason to do it like this? Does it allow logging something?